src/Morph/Web/select_overrides.js

```diff
@@ -0,0 +1,37 @@
+/**Workaround for https://github.com/ubports/morph-browser/issues/239 ( OTA-12 ) Until it is fixed in low level.
```

Comment:
No need for two stars in comments here, two stars marks the comment as a documentation comment. Documentation is not built from javascript files.